doc/release/upcoming_changes/19550.new_feature.rst

```diff
@@ -0,0 +1,12 @@
+Use f-strings to choose printing options for numeric dtypes
+-----------------------------------------------------------
+
+You can use f-strings now to format the precision, sign and the use of exponential notation for numeric arrays with a syntax inspired from [Python Format Specification Mini-Language](https://docs.python.org/3/library/string.html#formatspec).
```

Comment:
This might work since we are using rst not markup, let's see if CI builds it with a link
```suggestion
You can use f-strings now to format the precision, sign and the use of exponential
notation for numeric arrays with a syntax inspired from
:ref:py:`Python Format Specification Mini-Language <formatspec>`.
```